bitnami/spark/README.md

```diff
@@ -198,12 +198,15 @@ After that, your changes will be taken into account in the server's behaviour.
 
 ### Installing additional jars
 
-By default, this container bundles a generic set of jar files but the default image can be extended to add as many jars as needed for your specific use case. For instance, the following Dockerfile adds [`aws-java-sdk-bundle-1.11.704.jar`](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-bundle/1.11.704):
+By default, this container bundles a generic set of jar files but the default image can be extended to add as many jars as needed for your specific use case. For instance, the following Dockerfile adds [`aws-java-sdk-bundle-1.11.704.jar`](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-bundle/1.11.704) using a multi-stage Docker build:
 
 ```Dockerfile
+FROM curlimages/curl-base:latest as curl
```

Comment:
There is a straightforward example for adding `curl` at https://github.com/bitnami/containers/issues/13905#issuecomment-1317724282:

```
FROM bitnami/spark
USER root
RUN install_packages curl
USER 1001
RUN curl https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.11.704/aws-java-sdk-bundle-1.11.704.jar --output /opt/bitnami/spark/jars/aws-java-sdk-bundle-1.11.704.jar
```

In that case, a multi-stage build would not be necessary. Does it make sense to you to replace your example with this one?